### ISSUE TITLE
Feedback on Regression Project- Employee Burnrate Prediction

### ISSUE THREAD

**Issue by WamzMwangi** [graceobonyo/Employee-Burn-out-Rate-Prediction#1]
Hi Grace,

Here is feedback on the Employee Burnrate Prediction Problem

**What you did well**
1. You followed the correct machine learning workflow (though there are some unnecessary repetitions here and there)- for example in your workflow, after hyperparameter tuning you plot graphs comparing the model performance,  and had a very detailed EDA.
2. Good practice in using more than one regresssion metric to evaluate and compare across several models.
3.  Very clean use of ColumnTransformer + Pipeline

**Areas of improvement**

1. You failed to act on your own insight from your EDA- in the correlation heatmap you correctly identify that Joining Day and Joining Month are not strong predictors of employee burnout rate, yet you still fed them to the model- best to drop them since they are noise.
2. Use of hieristics without a clear explanation - its great calculating an overall score as a way to present a single value to compare models, but explain how and why you arrived at that particular formula for the composite score.
3. You choose to hypertune a RandomForest Regressor and then a few cells later, the XGBoost is your best performing model? Its not clear how we arrived at that- I see you  used the composite score to arrive at RandomForest and then change to R2 score to arrive at XGBoost, please keep your assumptions and workflow consistent.
3. You didn't tune your XGBoost...

**Future works**
1. Explainability and feature importance- what features are driving your predictions.
2. I liked that your used the predictions to rank the employees, and attached a burnrate to each employee- do some further investigations, what about high risk employees make them high risk?
